src/common/generator.js

```diff
@@ -21,9 +21,21 @@ window.POG=(function() {
             else {
                 selector += '[' + name + '=\'' + value + '\']';
             }
-            if (document.querySelectorAll(selector).length === 1) {
-                response = selector;
-            }
+	    //Surrounding with try to catch the exceptions that occur and crash the program on
+	    //certain web pages. 
+	    try
+	    {
+		if (document.querySelectorAll(selector).length === 1) {
+                    response = selector;
+		}
```

Comment:
Hi BogdanLivadariu. 

Can you tell me what you mean by the "Notify module". I am sure sure what you mean to specify. I can fix the indentation also.